backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/production/productionCandidate_with_BOM_recursion.feature

```diff
@@ -0,0 +1,176 @@
+@from:cucumber
+@ghActions:run_on_executor6
+Feature: Production dispo scenarios with BOMs whose components have their own BOMs in turn.
+
+  Background:
+    Given infrastructure and metasfresh are running
+    And the existing user with login 'metasfresh' receives a random a API token for the existing role with name 'WebUI'
+    And metasfresh has date and time 2024-09-20T08:00:00+01:00[Europe/Berlin]
+    And set sys config boolean value true for sys config SKIP_WP_PROCESSOR_FOR_AUTOMATION
+    And AD_Scheduler for classname 'de.metas.material.cockpit.stock.process.MD_Stock_Update_From_M_HUs' is disabled
+    And load M_Product_Category:
+      | M_Product_Category_ID   | Name     | Value    |
+      | standard_category_S0460 | Standard | Standard |
+    And create S_Resource:
+      | Identifier     | S_ResourceType_ID | IsManufacturingResource | ManufacturingResourceType | PlanningHorizon |
+      | resource_S0460 | 1000000           | Y                       | PT                        | 999             |
+    And metasfresh contains M_Warehouse:
+      | M_Warehouse_ID | PP_Plant_ID    |
+      | WH_S0460       | resource_S0460 |
```

Comment:
pls consider creating a new WH instead of changing an existing one
=> this will make the tests fragile if the other test is not configuring the plant

Reply:
agree, doing that